### PR TITLE
app-project: Add OrganizationLink to ProjectHeader and Hero components

### LINKED ISSUE #3579: Links to related organization from project home page
In PFE, if a project is part of an organization, then the project home page includes:
1. a link to the related organization above the project description (only on project home page)
2. a link to the related organization in the project sub-navigation after Recents (available on all project pages, i.e. About, Classify, etc.)

See https://www.zooniverse.org/projects/md68135/notes-from-nature-wedigflplants as an example.

An FEM project home page should have similar links to a related organization as PFE, if applicable.

### PR DESCRIPTION
**Pull request by mcbouslog**
## Package
- app-project

## Linked Issue and/or Talk Post
- closes #3579 

## Describe your changes
- add `fetchOrganization` helper
- refactor `getStaticPageProps` with `fetchOrganization`
- add organization store
- add OrganizationLink to ProjectHeader
- add OrganizationLink to Hero's NarrowLayout and WideLayout
- add organization link to DropdownNav

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - local
    - staging project with organization: https://local.zooniverse.org:3000/projects/markb-panoptes/uwworgtest-chicago-wildlife
    - production test project with translated organization title: https://local.zooniverse.org:3000/projects/fr/markbouslog/digileap-testing-production?env=production&demo=true
  - fe-project-branch
    - https://fe-project-branch.preview.zooniverse.org/projects/markb-panoptes/uwworgtest-chicago-wildlife?env=staging
    - https://fe-project-branch.preview.zooniverse.org/projects/fr/markbouslog/digileap-testing-production
- What user actions should my reviewer step through to review this PR?
  - note organization link in project header and project hero section
  - in [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats) note absence of organization links
  - in [production test project](https://local.zooniverse.org:3000/projects/fr/markbouslog/digileap-testing-production?env=production&demo=true) change language and note organization title change
- Which storybook stories should be reviewed?
  - http://localhost:9001/?path=/story/project-app-shared-project-header--organization-link
  - http://localhost:9001/?path=/story/project-app-screens-project-home-hero--with-organization
  - http://localhost:9001/?path=/story/project-app-shared-project-header-dropdown-nav--with-organization-link
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
- [ ] The PR creator has listed user actions to use when testing the new feature
- [ ] Unit tests are included for the new feature
- [ ] A storybook story has been created or updated
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

## Refactoring
- [ ] The PR creator has described the reason for refactoring
- [ ] The refactored component(s) continue to work as expected

## Post-merge
- [ ] This PR adds translations keys to English dictionary(s). See guidance for pulling new keys to Lokalise [here](https://github.com/zooniverse/how-to-zooniverse/blob/master/Translations/lokalise.md#lokalise-and-fem).

### CODE REVIEW

**Comment by mcbouslog** [comment]
I might combine this PR, #4569 and PR to Add OrganizationLink to Hero into one PR.

File: packages/app-project/src/components/ProjectHeader/components/OrganizationLink/OrganizationLink.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        top: 'medium'
+      }}
+    >
+      <StyledSpacedText
+        color='light-3'
+      >
+        {t('ProjectHeader.organization')}
+      </StyledSpacedText>
+      <StyledAnchor
+        href={`https://www.zooniverse.org/organizations/${slug}`}
```
Comment (mcbouslog) [inline packages/app-project/src/components/ProjectHeader/components/OrganizationLink/OrganizationLink.js:54]: This should change per ENV

File: packages/app-project/src/screens/ProjectHomePage/components/Hero/components/NarrowLayout/NarrowLayout.js [hunk 2]
```diff
@@ -23,6 +25,12 @@ function NarrowLayout (props) {
       <Grid margin={{ top: 'medium-neg', horizontal: 'medium' }}>
         <ContentBox gap='medium' >
           <Introduction />
+          {organization?.id ? (
+            <OrganizationLink
+              slug={organization.slug}
```
Comment (mcbouslog) [inline packages/app-project/src/screens/ProjectHomePage/components/Hero/components/NarrowLayout/NarrowLayout.js:30]: This needs to be updated to reflect ENV domain and `/organizations`

File: packages/app-project/src/screens/ProjectHomePage/components/Hero/components/OrganizationLink/OrganizationLink.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <Box
+      alignContent='start'
+      wrap={true}
+    >
+      <SpacedText>
+        {t('ProjectHeader.organization')}
+      </SpacedText>
+      <Anchor
+        href={`https://www.zooniverse.org/organizations/${slug}`}
```
Comment (mcbouslog) [inline packages/app-project/src/screens/ProjectHomePage/components/Hero/components/OrganizationLink/OrganizationLink.js:21]: Needs to be updated to reflect domain per ENV

File: packages/app-project/src/components/ProjectHeader/components/OrganizationLink/OrganizationLink.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/app-project/src/components/ProjectHeader/components/OrganizationLink/OrganizationLink.js:54]:
```suggestion
        href={`/organizations/${slug}`}
```

File: packages/app-project/src/screens/ProjectHomePage/components/Hero/components/OrganizationLink/OrganizationLink.js (hunk 3 above)
Comment (eatyourgreens) [reply packages/app-project/src/screens/ProjectHomePage/components/Hero/components/OrganizationLink/OrganizationLink.js:21]:
```suggestion
        href={`/organizations/${slug}`}
```

**Comment by mcbouslog** [comment]
Open item I need to follow-up on: showing organization link in project header when on classify and about pages.

File: packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js [hunk 4]
```diff
@@ -70,10 +71,16 @@ function ProjectHomePage ({
       <Media at='default'>
         <header>
           <ZooHeaderWrapper isAdmin={adminMode} />
-          <ProjectHeader adminMode={adminMode} />
+          <ProjectHeader
+            adminMode={adminMode}
+            organization={organization}
```
Comment (eatyourgreens) [inline packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js:76]: Out of interest, why isn't the organisation stored in the global store, like the project?

File: packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js [hunk 5]
```diff
@@ -125,11 +139,15 @@ function ProjectHomePage ({
 
 ProjectHomePage.defaultProps = {
   inBeta: false,
+  organization: {},
   workflows: []
 }
```
Comment (eatyourgreens) [inline packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js:144]: These should be moved into the `function` declaration. `defaultProps` is deprecated for functional components.

**Review by eatyourgreens — APPROVED** [review]
Looks good!

**Comment by eatyourgreens** [comment]
I’ve already approved this once. Please do feel free to merge it with changes.

**Comment by mcbouslog** [comment]
@seanmiller26 - these changes now include an organization link throughout a project's paths (home, classify, about) and in the DropdownNav with a small screen.

When you get a chance you can check it out at:
- https://fe-project-branch.preview.zooniverse.org/projects/markb-panoptes/uwworgtest-chicago-wildlife?env=staging
- https://fe-project-branch.preview.zooniverse.org/projects/fr/markbouslog/digileap-testing-production

Let me know what you think!

**Comment by seanmiller26** [comment]
The one note I'd point out is that "From the Organization: xyz" remains untranslated in different languages.

**Review by seanmiller26 — APPROVED** [review]
These changes look great!

**Pull request merged**